ui/docs/tutorial/authenticate.mdx

```diff
@@ -55,7 +55,7 @@ Credentials output as environment variables
 
 You can test the results with the same tool:
 
-```
+```json
```

Comment:
` ```bash `

Reply:
Sir, I'm done changing them.
thanks.

Reply:
Don't forget to push your changes so that I am able to see them.

Reply:
i have pushed them Sir